src/Common/src/System/Security/Cryptography/Asn1/AlgorithmIdentifierAsn.xml.cs

```diff
@@ -14,16 +15,16 @@ internal partial struct AlgorithmIdentifierAsn
     {
         internal Oid Algorithm;
         internal ReadOnlyMemory<byte>? Parameters;
-
+      
```

Comment:
nit: Added whitespace in a few places

Reply:
Ah, I see the same point raised here: https://github.com/dotnet/corefx/pull/40298/files#r313936711

Reply:
Yes, see above: https://github.com/dotnet/corefx/pull/40298#discussion_r313936711